XP/install/set-installation-defaults.ps1

```diff
@@ -41,32 +41,55 @@ $sql = $json.settings.sql
 $sql.server = "."
 $sql.adminUser = "sa"
 $sql.adminPassword = "12345"
+$sql.coreUser = $site.prefix + "coreuser"
+$sql.corePassword = "Test12345"
+$sql.masterUser = $site.prefix + "masteruser"
+$sql.masterPassword = "Test12345"
+$sql.webUser = $site.prefix + "webuser"
+$sql.webPassword = "Test12345"
+$sql.collectionUser = $site.prefix + "collectionuser"
+$sql.collectionPassword = "Test12345"
+$sql.reportingUser = $site.prefix + "reportinguser"
+$sql.reportingPassword = "Test12345"
+$sql.processingPoolsUser = $site.prefix + "poolsuser"
+$sql.processingPoolsPassword = "Test12345"
+$sql.processingTasksUser = $site.prefix + "tasksuser"
+$sql.processingTasksPassword = "Test12345"
+$sql.referenceDataUser = $site.prefix + "referencedatauser"
+$sql.referenceDataPassword = "Test12345"
+$sql.marketingAutomationUser = $site.prefix + "marketingautomationuser"
+$sql.marketingAutomationPassword = "Test12345"
+$sql.formsUser = $site.prefix + "formsuser"
+$sql.formsPassword = "Test12345"
+$sql.exmMasterUser = $site.prefix + "exmmasteruser"
+$sql.exmMasterPassword = "Test12345"
+$sql.messagingUser = $site.prefix + "messaginguser"
+$sql.messagingPassword = "Test12345"
```

Comment:
See other suggested changes in overrides file to assignment passwords using a (common) variable. Suggest the same is done here.